### ISSUE TITLE
Guestbook does not inherit custom CSS styles

### ISSUE THREAD

**Issue by cesarferradas** [standardnotes/listed#28]
### Steps to reproduce

1. Add a custom `styles.css` file and publish to your blog as describer [here](https://docs.standardnotes.org/listed/how-do-i-change-the-colors-fonts-and-general-layout-of-my-listed-blog)
2. See that the blog shows with new styling.
3. Go to the "Guestbook" page, and notice the styling is gone.

### Potential solution

Looking at the source code, it seems that the styling is not added as it should here: https://github.com/standardnotes/listed/blob/7123366a670ea1bb91fdcec5d66de4dc11530a63/app/views/layouts/application.html.erb#L52

Presumably because `@styles` is not being set.

In other pages where styling is applied e.g. the Posts pages, you set the styles in the controller:

https://github.com/standardnotes/listed/blob/8c621769acdc391539dca1d8adb00d364f168e69/app/controllers/posts_controller.rb#L94

Would the fix be as simple as adding:

```ruby
@styles = @author.styles
```

to the Guestbook controller in the `before_action` block?

https://github.com/standardnotes/listed/blob/a3139a63d509a47d174d4d567cddbedbe5c7efc0/app/controllers/guestbook_entries_controller.rb#L4

Let me know and I'm happy to make a PR.

**Comment by atmoio** [comment]
> Would the fix be as simple as adding:

Seems like it! Perhaps we can add this to any author page that should be stylized and remove it from being exclusive to posts_controller.show method.

**Comment by cesarferradas** [comment]
Sounds good! Thanks for clarifying.

Looks like we already add the styles and meta images to all author pages in the `authors_controller`:

https://github.com/standardnotes/listed/blob/c1e5f4799cebc736ed1a7f5760deaca122d471f8/app/controllers/authors_controller.rb#L16

But I don't think the parent route's `before_action` runs for the sub-routes?

That's the only way I can explain the nested `:guestbook` path being the only one with no custom CSS or favicon. The nested `:posts` path required adding this to the `posts_controller` so that it didn't have the same issue (I think?):

```ruby
@styles = @post.author.styles
set_meta_images_for_author(@post.author)
```

Even though similar code exists in the parent `authors_controller`.

I think a quick win would be to just add a similar code block to the `guestbook_entries_controller`'s `before_action`.

Will make a PR, let me know if you're happy with the proposed changes.

**Comment by atmoio** [comment]
> But I don't think the parent route's before_action runs for the sub-routes?

Tbh you'll just have to run and test the different scenarios, as it's been well over a year since I've been in this codebase, so can't really remember specifics.

> a quick win

😵Quick win == technical debt.

But feel free to create the PR in the direction you would naturally take, and then we could discuss specifics on the PR itself.

**Comment by antsgar** [comment]
This one's probably been fixed with #68
